x/marker/types/marker_test.go

```diff
@@ -108,95 +108,95 @@ func TestNewMarkerValidate(t *testing.T) {
 		},
 		{
 			"insufficient supply",
-			NewMarkerAccount(baseAcc, sdk.NewCoin("test", sdk.ZeroInt()), manager, nil, StatusFinalized, MarkerType_Coin, true, true, false),
+			NewMarkerAccount(baseAcc, sdk.NewCoin("test", sdk.ZeroInt()), manager, nil, StatusFinalized, MarkerType_Coin, true, false, false, []string{}),
```

Comment:
Is there a reason to change the `allowGovernanceControl` value in these tests?

Reply:
Nope, did it by accident while fixing merge.  Changing it back.